frontend/cypress/tests/codeView/codeView_renderFilterHash.cy.js

```diff
@@ -309,31 +309,184 @@ describe('render filter hash', () => {
     cy.url()
       .should('not.contain', 'gradle');
 
+    cy.url()
+      .should('not.contain', 'yml');
+
```

Comment:
Can we make it clear why gradle and yml were the 2 file types selected here with a comment?
For example,
```
// Assumption: gradle is the first file type and yml is the last file type